cluster/src/main/scala/io/snappydata/ToolsCallbackImpl.scala

```diff
@@ -209,4 +215,19 @@ object ToolsCallbackImpl extends ToolsCallback with Logging {
 
   override def checkSchemaPermission(schema: String, currentUser: String): String =
     StoreCallbacksImpl.checkSchemaPermission(schema, currentUser)
+
+  override def removeURIs(uris: Array[String], isPackage: Boolean): Unit = {
+    val lead = ServiceManager.getLeadInstance.asInstanceOf[LeadImpl]
+    val allURLs = lead.urlclassloader.getURLs
+    val updatedURLs = allURLs.toBuffer
+    uris.foreach(uri => {
+      val newUri = new URI("file:" + uri)
+      if (updatedURLs.contains(newUri.toURL)) {
+        updatedURLs.remove(updatedURLs.indexOf(newUri.toURL))
+      }
+    })
+    lead.urlclassloader = new ExtendibleURLClassLoader(lead.urlclassloader.getParent)
```

Comment:
Please test the case where a secondary lead is brought in after a system is up and then lead becomes the primary leader. ( Not for this merge, but please note this )